packages/contracts/test/integration/eth-virtual-app-agreement.spec.ts

```diff
@@ -0,0 +1,149 @@
+import * as ethers from "ethers";
+
+import * as Utils from "../../utils/misc";
+
+const web3 = (global as any).web3;
+const { provider, unlockedAccount } = Utils.setupTestEnv(web3);
+
+import { AbstractContract, buildArtifacts, expect } from "../../utils";
+
+/// Deploys a new DelegateProxy instance, funds it, and delegatecalls to
+/// FixedResolutionApp with random beneficiaries
+const delegatecallVirtualAppAgreement = async (
+  virtualAppAgreement: ethers.Contract,
+  registry: ethers.Contract,
+  resolutionAddr: string,
+  expiry: number,
+  capitalProvided: ethers.utils.BigNumber,
+  assetType: number
+): Promise<string[]> => {
+  const delegateProxy = await (await AbstractContract.fromArtifactName(
+    "DelegateProxy"
+  )).deploy(unlockedAccount);
+
+  await unlockedAccount.sendTransaction({
+    to: delegateProxy.address,
+    value: ethers.utils.parseEther("100")
```

Comment:
style: We're moving towards importing things like `parseEther` at module level `import { parseEther } from "ethers/utils";` and do on for the rest of this file

Reply:
Can we make that change elsewhere then?